charts/goldpinger/Chart.yaml

```diff
@@ -10,4 +10,5 @@ maintainers:
 name: goldpinger
 sources:
   - https://github.com/bloomberg/goldpinger
-version: 2.0.2
+  - https://github.com/okgolove/helm-charts
+version: 2.0.3
```

Comment:
I think we should bump major version here.

Reply:
ok. updated version charts